rts/lra/lra-coordinator-jar/pom.xml

```diff
@@ -103,6 +103,12 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>jakarta.enterprise</groupId>
+            <artifactId>jakarta.enterprise.cdi-api</artifactId>
```

Comment:
I don't think this dependency should be required now? Or at least javax/annotation is not provided directly inside it. If a dependency is required I suspect it would be on one from https://search.maven.org/artifact/jakarta.annotation/jakarta.annotation-api

Reply:
+1